### PR TITLE
13.0 mig partner contact birthdate

### PR DESCRIPTION
**Pull request by jeroenev**

### CODE REVIEW

File: partner_contact_birthdate/README.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fpartner--contact-lightgray.png?logo=github
+    :target: https://github.com/OCA/partner-contact/tree/12.0/partner_contact_birthdate
+    :alt: OCA/partner-contact
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/partner-contact-12-0/partner-contact-12-0-partner_contact_birthdate
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/134/12.0
```
Comment (Daemo00) [inline partner_contact_birthdate/README.rst]: Please regenerate the readme using branch `13.0`

**Review by Daemo00 — CHANGES_REQUESTED** [review]
Thanks! Just a little note on the readme

File: partner_contact_birthdate/README.rst (hunk 1 above)
Comment (jeroenev) [reply partner_contact_birthdate/README.rst]: regenerated

File: partner_contact_birthdate/readme/INSTALLATION.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+To install this module, you need to:
```
Comment (Daemo00) [inline partner_contact_birthdate/readme/INSTALLATION.rst]: Should be named `INSTALL.rst`

File: partner_contact_personal_information_page/README.rst [hunk 3]
```diff
@@ -0,0 +1,91 @@
+======================================
```
Comment (Daemo00) [inline partner_contact_personal_information_page/README.rst]: Migration of this one is already managed in https://github.com/OCA/partner-contact/pull/788

**Review by Daemo00 — COMMENTED** [review]
Thanks, maybe we should wait for https://github.com/OCA/partner-contact/pull/788 to be merged, then rebase on `13.0` and then have a proper look to this with the help of the bots.
Anyway this already looks good

File: partner_contact_personal_information_page/README.rst (hunk 3 above)
Comment (jeroenev) [reply partner_contact_personal_information_page/README.rst]: Yes, I migrated both
This branch is rebased on that one

**Review by Daemo00 — APPROVED** [review]
Thanks! Just code review, looks good to me.
In order to clean commit history/files changed of this PR, I recommend to rebase on `13.0`.

File: partner_contact_birthdate/views/res_partner.xml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+     Copyright 2015 Grupo ESOC Ingeniería de Servicios, S.L.U. -->
+
+<odoo>
+    <record id="view_personal_information_birthdate_date" model="ir.ui.view">
```
Comment (NL66278) [inline partner_contact_birthdate/views/res_partner.xml:6]: id should be the same as the inherit_id (without the module name of course).

**Review by NL66278 — CHANGES_REQUESTED** [review]
:+1: Mostly good. One nag.

**Comment by newtratip** [comment]
This PR is duplicated with https://github.com/OCA/partner-contact/pull/833 or not.

**Review by NL66278 — APPROVED** [review]
:+1: Tolerating the id differing from the inherit_id in view inheritance for the moment.

**Review by HaraldPanten — CHANGES_REQUESTED** [review]
This module was already merged in V13 branch for this repository

**Pull request closed without merging**